salt/edx/mitx_etl.sls

```diff
@@ -0,0 +1,57 @@
+install_python3_pip:
+  pkg.installed:
+    - pkgs:
+        - python3
+        - python3-pip
+    - refresh: True
+    - require:
+      - pkg: install_python_3
+
+create_mitx_directory:
+  file.directory:
+    - name: /mitx
+
+clone_mitx_etl_repo:
+  git.latest:
+    - name: https://github.com/mitodl/odl-etl
+    - target: /mitx
+    - force_clone: True
+    - require:
+      - pkg: install_python_3
+
+install_mitx_residential_etl_requirements:
+  virtualenv.managed:
+    - name: /mitx/mitx_etl
+    - system_site_packages: False
+    - requirements: /mitx/requirements.txt
+    - env_vars:
```

Comment:
This is missing a specification of the python version to use to build the virtualenv, so it will likely default to Python2 which is the system default (I believe)

Reply:
Added path to python3 executable